.github/workflows/pr-checks.yml

```
@@ -0,0 +1,87 @@
+            --exclude "site\.webmanifest"
+            "./out/**/*.html"
+          fail: true
+          failIfEmpty: true
+
+      - name: Deploy Cloudflare Pages preview
+        id: deploy-preview
+        if: github.event.pull_request.head.repo.full_name == github.repository
+        env:
+          CLOUDFLARE_API_TOKEN: ${{ secrets.CLOUDFLARE_API_TOKEN }}
+          CLOUDFLARE_ACCOUNT_ID: ${{ secrets.CLOUDFLARE_ACCOUNT_ID }}
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)